app/services/users.js

```diff
@@ -0,0 +1,27 @@
+const { User } = require('../models');
+const errors = require('../errors');
+
+const getUserByEmail = email => {
```

Comment:
I would change this for `exports.getUserByEmail`

Reply:
export mode changed